ewkcoffea/modules/keynote_tools.py

```diff
@@ -0,0 +1,594 @@
+#!/bin/env python
```

Comment:
So if this is not used (which I think you mentioned on mattermost), it should be removed, right?